#! /bin/sh /usr/share/dpatch/dpatch-run
## 03_support_nrpe.d.dpatch by Alexander Wirt <formorer@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Support an nrpe.d directory

@DPATCH@
diff -urNad nagios-nrpe-2.8.1~/sample-config/nrpe.cfg.in nagios-nrpe-2.8.1/sample-config/nrpe.cfg.in
--- nagios-nrpe-2.8.1~/sample-config/nrpe.cfg.in	2008-04-15 23:53:48.000000000 +0200
+++ nagios-nrpe-2.8.1/sample-config/nrpe.cfg.in	2008-04-15 23:54:48.000000000 +0200
@@ -211,3 +211,9 @@
 # local configuration:
 #	if you'd prefer, you can instead place directives here
 include=/etc/nagios/nrpe_local.cfg
+
+# 
+# you can place your config snipplets into nrpe.d/
+include_dir=/etc/nagios/nrpe.d/
+
+
